python/cog/command/ast_openapi_schema.py

```diff
@@ -307,8 +309,25 @@ def find(obj: ast.AST, name: str) -> ast.AST:
     """Find a particular named node in a tree"""
     return next(node for node in ast.walk(obj) if getattr(node, "name", "") == name)
 
+if typing.TYPE_CHECKING:
+    AstVal: "typing.TypeAlias" = int | float | complex | str | list["AstVal"] | bytes | None
+    AstValNoBytes: "typing.TypeAlias" = int | float | str | list["AstValNoBytes"]
+    JSONObject: "typing.TypeAlias" = int | float | str | list["JSONObject"] | "JSONDict" | None
+    JSONDict: "typing.TypeAlias" = dict[str, "JSONObject"]
+
```

Comment:
I think this breaks on 3.8

Reply:
also, wow, a true JSONDict type, intense, I had always given up on doing recursive types like this in the past